stardis/io/model/tests/test_model_io.py

```diff
@@ -8,57 +8,53 @@
 
 from stardis.io.model.marcs import read_marcs_model
 
-
 @pytest.fixture
-def marcs_model_test_data_file_path():
+def create_model():
     fname = "marcs_test.mod.gz"
     fpath = files("stardis.io.model.tests.data")
-    return fpath.joinpath(fname)
+    test_data_filepath= fpath.joinpath(fname)
```

Comment:
This isn't compliant with black formatting. Maybe go grab the black formatting vscode extension to automatically format your code when you save the file.